docs/source/install/kernel_install.rst

```diff
@@ -53,3 +53,33 @@ For example, using conda environments:
 The ``--name`` value is used by Jupyter internally. These commands will overwrite
 any existing kernel with the same name. ``--display-name`` is what you see in
 the notebook menus.
+
+Using virtualenv you can add your new environments by linking to specific instance of your Python:
+
+.. sourcecode:: bash
+
+    python -m ipykernel install --prefix=/path/to/new_env --name python_new_env
+  
+Note that this command will create new configuration for kernel in one of it's prefered location (see ``jupyter --paths`` command for more details):
+
+* system-wide (e.g. /usr/local/share),
+* in Jupyter's env (sys.prefix/share),
+* per-user (~/.local/share or ~/Library/share)
+
+You can also create configuration in temporary location and install it in Jupyter (copy configuration files) with:
```

Comment:
For this case, I think it only makes sense to do this if you want to edit the kernelspec by hand before installing it. I would add that to the discussion of this one, or remove the more complicated two-step case.